collectionDBConnection.php

```diff
@@ -0,0 +1,9 @@
+<?php
+
+$db = new PDO ('mysql:host=db; dbname=OfficeQuotesDB', 'root', 'password');
+$db -> setAttribute(PDO::ATTR_DEFAULT_FETCH_MODE, PDO::FETCH_ASSOC);
+$query = $db->prepare('SELECT * FROM `quotes` INNER JOIN `images` ON `quotes`.`id` = `images`.`quotes-ID`');
+$query->execute();
+$quotes = $query->fetchAll();
```

Comment:
And I'd also suggest wrapping this in a function that takes a PDO connection as a parameter, and returns the array of results.

It could then be called in your main collection.php file when needed.